Gruntfile.js

```diff
@@ -5,26 +5,24 @@ module.exports = function(grunt) {
 
   var tsConfig = {
     dev: {
-      src: ["src/**/*.ts", "typings/**/*.d.ts"],
-      outDir: "build/src/",
+      src: ["src/**/*.ts"],
+      out: "plottable.js",
       options: {
         target: "es5",
         noImplicitAny: true,
         sourceMap: false,
         declaration: true,
-        compiler: "./node_modules/typescript/bin/tsc",
```

Comment:
If we delete this line, we don't even need to specify `typescript` as a dependency in our `package.json`, as it will always use `grunt-ts` dependency all the time